.github/workflows/main.yml

```
@@ -66,3 +66,20 @@ jobs:
           path: |
             test-suite/tests/test*/test.err*
             test-suite/tests/test*/test.out*
+
+  docs:
+    name: Validate mkdocs links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          # use the latest stable version
+          python-version: "3.x"
+          cache: "pip"
+      - run: pip install -r docs/requirements.txt
```

Oh great mkdocs is hit by the same issue that `sphinxcontrib-bibtex` has: https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/345

```suggestion
     # `pybtex` uses `pkg_resources` which is deprecated. Use workaround until upstream `mkdocs_bibtext`decides on a solution
     # https://bitbucket.org/pybtex-devs/pybtex/issues/169/replace-pkg_resources-with
      - run: pip install -r docs/requirements.txt setuptools
```

---

Here's another issue you can follow: https://github.com/shyamd/mkdocs-bibtex/issues/228

---

Indeed, now I remember why I fix the python version to `3.11` in the yaml.
And thanks for creating an issue in the `mkdocs-bibtex` repo!